mw-webapp/src/service/GoalService.ts

```diff
@@ -32,6 +32,22 @@ export class GoalService {
     return validatedGoalsDTO;
   }
 
+  /**
+   * Get GoalsDTO by uuid
+   */
+  public static async getGoalsDTOByUuids(goalUuids: string[]): Promise<GoalDTO[]> {
+    const goalsRef = collection(db, PATH_TO_GOALS_COLLECTION);
```

Comment:
looks like we have limit for firebase 30 entities maximum, lets use approach like in getSortedDayReportsDTO, but we need to build general helper, check it

Reply:
Copied approach